README.md

```diff
@@ -7,6 +7,14 @@ This project contains the Cylc UI Server. A JupyterHub-compatible application,
 used to display the Cylc UI (or simply UI) to users, and to communicate with
 Workflow Services (WFS).
 
+[Cylc Website](https://cylc.org/) |
+[Contributing](CONTRIBUTING.md)
+
+## Contents
+
+[Installation](#installation)
```

Comment:
Note, as this one doesn't have an extra blank line, nor the `|` character as above, it may appear that both are actually a single link, until user moves mouse over. Everything else looks good.

Reply:
Oops, good spotting.

Reply:
Fixed.